client/state/selectors/should-sync-reader-follows.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Internal Dependencies
+ */
+import { getReaderFollowsLastSyncTime } from 'state/selectors';
+
+const MS_BETWEEN_SYNCS = 1000 * 60 * 60; // one hour
+
+export default function shouldSyncReaderFollows( state ) {
+	const lastTime = getReaderFollowsLastSyncTime( state );
```

Comment:
whats the general best-practice for selectors using selectors?
it feels okay to me